developer_manual/_static/openapi.json

```diff
@@ -0,0 +1,16975 @@
+{
```

Comment:
This file seems to be the build asset of the merge step, right? Just double checking if we actually should commit it or rely on always building?

Reply:
Ah yeah better to not commit it.